### ISSUE TITLE
Netdata in Docker cannot load stock config (permission denied)

### ISSUE THREAD

**Issue by ccremer** [netdata/netdata#4453]
##### Bug report summary
```
2018-10-20 23:03:40: netdata INFO  : MAIN : CONFIG: cannot load stock config '/usr/lib/netdata/conf.d/stream.conf'. Running with internal defaults.
2018-10-20 23:03:40: netdata ERROR : MAIN : Health configuration cannot read file '/usr/lib/netdata/conf.d/health.d/apcupsd.conf'. (errno 13, Permission denied)
2018-10-20 23:03:40: netdata ERROR : MAIN : Health configuration cannot read file '/usr/lib/netdata/conf.d/health.d/zfs.conf'. (errno 13, Permission denied)
2018-10-20 23:03:40: netdata ERROR : MAIN : Health configuration cannot read file '/usr/lib/netdata/conf.d/health.d/fronius.conf'. (errno 13, Permission denied)
2018-10-20 23:03:40: netdata ERROR : MAIN : Health configuration cannot read file '/usr/lib/netdata/conf.d/health.d/bind_rndc.conf'. (errno 13, Permission denied)
... and tons more
```
This ultimately results in no alarms being loaded, and probably other things.

###### OS / Environment
docker, latest `netdata/netdata:latest-amd64` pull, image ID `fbef9f48cbb7`

##### Component Name
core netdata (config)

##### Steps To Reproduce
Run the following docker-compose file:

```yaml
version: '2'
services:
  netdata:
    restart: always
    image: netdata/netdata:latest-amd64
    #container_name: netdata
    #hostname: netdata
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock:ro
      - /etc/localtime:/etc/localtime:ro
      - /sys:/host/sys
      - /proc:/host/proc
      #- ./stream.conf:/etc/netdata/stream.conf
    ports:
      - "19990:19999"
```
Then, watch the logs:
`docker logs <container_name> 2>&1 | grep /usr/lib/netdata`

A `ls -RAhl /usr/lib/netdata/*` within the container shows that every file is owned by `root:root`, with mode `-rw-r-----` (640), and directories with `drwxr-xr-x` (755)

##### Expected behavior
Loaded alarms upon activating fping plugin (in my case)

##### Proposed fix
Either run
`chown -R netdata:root /usr/lib/netdata/*`
or
`chmod -R a+r /usr/lib/netdata/*`
in the Dockerfile, whatever you feel more comfortable security-wise. Somewhere near whare the other permissions are applied. Both work, I have tested it in a derived Dockerfile.

**Comment by paulfantom** [comment]
Should be fixed in #4454.

FYI you don't need to use `latest-amd64` tag, `latest` should do the CPU architecture autodetection and choose the same image.

**Comment by ccremer** [comment]
Nice!
Actually, `chown -R root:netdata /usr/lib/netdata` works as well, instead of `chown -R netdata:root /usr/lib/netdata` (notice group/user switch)
